spec/sample-data/valid/control-statement-02.json

```diff
@@ -0,0 +1,48 @@
+
+{
+  "subject_entity": {
+    "entity_properties": {
+      "company_number": "01589850",
```

Comment:
Is it deliberate that this number is the same as the controlling one below? If not, and assuming it's not expected in source data, it would be good to change it to prevent confusion. Nice to change to `09876543` to match the other examples.